types/office-js-preview/index.d.ts

```diff
@@ -476,6 +516,114 @@ declare namespace Office {
         */
         value: T;
     }
+    /**
+     * Message used in the visibility mode changed event.
+     */
+    interface VisibilityModeChangedMessage {
+        /**
+         * Visibility changed state.
+         */
+        visibilityMode: Office.VisibilityMode;
+    }
+    /**
+     * Function type to turn off the event.
```

Comment:
"the event" - which event is that?

Reply:
It's a generic type and can be used across any events that returns this function.
this is to avoid having to need the object to call the event off method.

Reply:
Then shouldn't it be "an event" instead of "the event"?